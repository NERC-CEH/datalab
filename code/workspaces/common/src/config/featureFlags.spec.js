import featureFlagsConfig from './featureFlags';

describe('featureFlagsConfig', () => {
  it('has expected content', () => {
    expect(featureFlagsConfig()).toMatchSnapshot();
  });
});
