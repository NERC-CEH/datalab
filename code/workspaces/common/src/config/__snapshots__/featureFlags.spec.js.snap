// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`featureFlagsConfig has expected content 1`] = `
Object {
  "expireUnusedNotebooks": Object {
    "accessTimeLimit": 31,
    "accessTimeWarning": 14,
    "inUseTimeWarning": 8,
  },
  "requestProjects": false,
}
`;
